backend/helpers/config.ts

```diff
@@ -5,7 +5,7 @@ import { Config, FeedbackConfig } from '../types/config';
 dotenv.config({ path: path.resolve(process.cwd(), '.env') });
 
 const loadConfig = (): Config => {
-  const provider = process.env.VIDEO_SERVICE_PROVIDER ?? 'opentok';
+  const provider = process.env.VIDEO_SERVICE_PROVIDER ?? 'vonage';
```

Comment:
Looks like this broke the integration tests:
```
[WebServer] /home/runner/work/vonage-video-react-app/vonage-video-react-app/backend/helpers/config.ts:24
      throw new Error('Missing config values for Vonage');
```
https://github.com/Vonage/vonage-video-react-app/actions/runs/12998700535/job/36252401556?pr=33